docs/tests/system_checklist.rst

```diff
@@ -160,13 +160,12 @@ Checklist
 
       To check for errors, grep that for "ERROR".
 
-    * Run QA Selenium tests.
+    * Run QA Selenium WebDriver tests.
 
-      localdev: ?
-      stage: In IRC: "webqatestbot build socorro.stage.saucelabs"
-      prod: In IRC: "webqatestbot build socorro.prod.saucelabs"
+      See the README in the /e2e-tests/ directory, here:
+      https://github.com/mozilla-services/socorro/blob/master/e2e-tests/README.md
```

Comment:
The README for the e2e tests is pulled in to our Sphinx build of the main Socorro docs and is available on RTD: http://socorro.readthedocs.io/en/latest/tests/e2e_readme.html#set-up-and-run-socorro-tests

Maybe just add a colon after `tests` and replace the `See the README` paragraph with that link?